flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -119,7 +121,7 @@ public void after() {
     //  Tests for keyed ValueState
     // -------------------------------------------------------------------------------
 
-    @Test
+    @TestTemplate
     public void testKeyedValueStateMigration() throws Exception {
```

Comment:
```suggestion
    void testKeyedValueStateMigration() throws Exception {
```
Could be package private